docs/csharp/methods.md

```diff
@@ -34,7 +34,7 @@ This topic contains the following sections:
 - [Expression-bodied members](#expr)
 - [Iterators](#iterators)
 
-<a name="signatures" /a>
+<a name="signatures" />
```

Comment:
GuardRex - I suggested your approach (closing tags) to Maira originally, because I thought that was always the best rule, but she preferred the self-closing tags for this file - which is why I did it that way.